### PR TITLE
meta: move mmarchini to emeritus

### PR DESCRIPTION
**Pull request by mmarchini**
Ref: https://github.com/nodejs/node/pull/42750

### CODE REVIEW

**Review by mcollina — APPROVED** [review]
lgtm

**Review by Trott — APPROVED** [review]

**Comment by Trott** [comment]
Should we get rid of the Calendar Maintainers Emeriti list? Not sure there's a lot of value to that.

**Review by aduh95 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -97,6 +96,7 @@ This is a list of former maintainers of the Node.js Community Calendar. If these
 - [@bamieh](https://github.com/bamieh) - **Ahmad Bamieh**
 - [@dshaw](https://github.com/dshaw) - **Dan Shaw**
 - [@maddhruv](https://github.com/maddhruv) - **Dhruv Jain**
+- [@mmarchini](https://github.com/mmarchini) - **Mary Marchini**
 - [@gibfahn](https://github.com/gibfahn) - **Gibson Fahnestock**
 - [@hackygolucky](https://github.com/hackygolucky) - **Tracy Hinds**
 - [@williamkapke](https://github.com/williamkapke) - **William Kapke**
```
Comment (Trott) [inline README.md:102]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -94,9 +93,3 @@ This list should be reviewed and pruned annually (at minimum). The calendar has
 This is a list of former maintainers of the Node.js Community Calendar. If these members become active in the project again and have a reason to be a maintainer, they are more than welcome to move themselves to the [Calendar Maintainers](#calendar-maintainers) list above via a Pull Request.
 
 - [@amiller-gh](https://github.com/amiller-gh) - **Adam Miller**
```
Comment (Trott) [inline README.md:95]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -84,19 +84,9 @@ This list should be reviewed and pruned annually (at minimum). The calendar has
 - [@joesepi](https://github.com/joesepi) - **Joe Sepi**
 - [@mcollina](https://github.com/mcollina) - **Matteo Collina**
 - [@mhdawson](https://github.com/mhdawson) - **Michael Dawson**
-- [@mmarchini](https://github.com/mmarchini) - **Mary Marchini**
 - [@MylesBorins](https://github.com/MylesBorins) - **Myles Borins**
 - [@ryanmurakami](https://github.com/ryanmurakami) - **Ryan Lewis**
 - [@trott](https://github.com/trott) - **Rich Trott**
 
 ### Emeriti Calendar Maintainers
```
Comment (Trott) [inline README.md:92]:
```suggestion
```

File: README.md [hunk 4]
```diff
@@ -84,19 +84,7 @@ This list should be reviewed and pruned annually (at minimum). The calendar has
 - [@joesepi](https://github.com/joesepi) - **Joe Sepi**
 - [@mcollina](https://github.com/mcollina) - **Matteo Collina**
 - [@mhdawson](https://github.com/mhdawson) - **Michael Dawson**
-- [@mmarchini](https://github.com/mmarchini) - **Mary Marchini**
 - [@MylesBorins](https://github.com/MylesBorins) - **Myles Borins**
 - [@ryanmurakami](https://github.com/ryanmurakami) - **Ryan Lewis**
 - [@trott](https://github.com/trott) - **Rich Trott**
```
Comment (Trott) [inline README.md:90]:
```suggestion
```

**Review by Trott — APPROVED** [review]

**Pull request merged**